iphone/Classes/TiUIListView.m

```diff
@@ -1815,7 +1815,32 @@ - (void)scrollViewWillBeginDragging:(UIScrollView *)scrollView
         [self.proxy fireEvent:@"dragstart" withObject:nil withSource:self.proxy propagate:NO reportSuccess:NO errorCode:0 message:nil];
     }
 }
+-(void)scrollViewWillEndDragging:(UIScrollView *)scrollView withVelocity:(CGPoint)velocity targetContentOffset:(inout CGPoint *)targetContentOffset{
+    
+    if ([self.proxy _hasListeners:@"scrolling"]) {
+        NSString* directionString = @"none";
+
+        if (velocity.y > 0) {
+            directionString = @"up";
+        }
 
+        if (velocity.y < 0) {
+            directionString = @"down";
+        }
+        
+        if(directionString != @"none") {
+            NSMutableDictionary *eventArgs = [NSMutableDictionary dictionary];
+            NSDictionary *pointObject = [NSDictionary dictionaryWithObjectsAndKeys:
+             [NSNumber numberWithDouble:targetContentOffset->x],@"x",
+             [NSNumber numberWithDouble:targetContentOffset->y],@"y",
+                                         nil];
+            [eventArgs setValue:directionString forKey:@"direction"];
+            [eventArgs setValue:NUMDOUBLE(velocity.y) forKey:@"velocity"];
+            [eventArgs setObject:pointObject forKey:@"targetContentOffset"];
+            [self.proxy fireEvent:@"scrolling" withObject:eventArgs withSource:self.proxy propagate:NO reportSuccess:NO errorCode:0 message:nil];
```

Comment:
You need to check if the event-listener is added beforehand. Will also mention it in my gist.